src/main/java/com/haxwell/apps/questions/entities/Difficulty.java

```diff
@@ -23,10 +23,13 @@
 import java.util.Set;
 
 import javax.persistence.Entity;
-import javax.persistence.Id;
+//import javax.persistence.Id;
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
+//This class did not implement javax.persistance.GenerationType using it now with ATE might cause conflicts in database
+

```

Comment:
Yes, that is because we do not generate new Difficulty objects. There is a defined set of them, and we only use those. Question, Choice, Exam, Vote, etc.. we generate new ones of those.